### ISSUE TITLE
"System.Exception: Magic number is wrong: 542" with duplicati-cli on Antergos Linux

### ISSUE THREAD

**Issue by blacky14** [duplicati/duplicati#3078]
- [x] I have searched open and closed issues for duplicates.

----------------------------------------

## Environment info

- **Duplicati version**: 2.0.2.20
- **Operating system**: Antergos Linux
- **Backend**: Backblaze B2

## Description

I'm trying to execute the duplicati-cli to do some backup tests, but it always terminates with a "System.Exception: Magic number is wrong: 542".

## Steps to reproduce

Call duplicati-cli with any parameters:

```
duplicati-cli 

Unhandled Exception:
System.TypeInitializationException: The type initializer for 'System.Console' threw an exception. ---> System.TypeInitializationException: The type initializer for 'System.ConsoleDriver' threw an exception. ---> System.Exception: Magic number is wrong: 542
  at System.TermInfoReader.ReadHeader (System.Byte[] buffer, System.Int32& position) [0x00028] in <a84b655e5e6a49ee96b338ec792f5580>:0 
  at System.TermInfoReader..ctor (System.String term, System.String filename) [0x0005f] in <a84b655e5e6a49ee96b338ec792f5580>:0 
  at System.TermInfoDriver..ctor (System.String term) [0x00055] in <a84b655e5e6a49ee96b338ec792f5580>:0 
  at System.ConsoleDriver.CreateTermInfoDriver (System.String term) [0x00000] in <a84b655e5e6a49ee96b338ec792f5580>:0 
  at System.ConsoleDriver..cctor () [0x0004d] in <a84b655e5e6a49ee96b338ec792f5580>:0 
   --- End of inner exception stack trace ---
  at System.Console.SetupStreams (System.Text.Encoding inputEncoding, System.Text.Encoding outputEncoding) [0x00007] in <a84b655e5e6a49ee96b338ec792f5580>:0 
  at System.Console..cctor () [0x0008e] in <a84b655e5e6a49ee96b338ec792f5580>:0 
   --- End of inner exception stack trace ---
  at System.Diagnostics.Process.StartWithCreateProcess (System.Diagnostics.ProcessStartInfo startInfo) [0x00337] in <640558357cc84390be8186623166329d>:0 
  at System.Diagnostics.Process.Start () [0x0003a] in <640558357cc84390be8186623166329d>:0 
  at (wrapper remoting-invoke-with-check) System.Diagnostics.Process:Start ()
  at System.Diagnostics.Process.Start (System.Diagnostics.ProcessStartInfo startInfo) [0x0001b] in <640558357cc84390be8186623166329d>:0 
  at Duplicati.Library.AutoUpdater.UpdaterManager.RunFromMostRecentSpawn (System.Reflection.MethodInfo method, System.String[] cmdargs, Duplicati.Library.AutoUpdater.AutoUpdateStrategy defaultstrategy) [0x00196] in <bea267acd22e46748297665d861d8f02>:0 
  at Duplicati.Library.AutoUpdater.UpdaterManager.RunFromMostRecent (System.Reflection.MethodInfo method, System.String[] cmdargs, Duplicati.Library.AutoUpdater.AutoUpdateStrategy defaultstrategy) [0x0001b] in <bea267acd22e46748297665d861d8f02>:0 
  at Duplicati.CommandLine.Program.Main (System.String[] args) [0x0001a] in <cce58b5528b54025a8c535610dc1cdaf>:0 
[ERROR] FATAL UNHANDLED EXCEPTION: System.TypeInitializationException: The type initializer for 'System.Console' threw an exception. ---> System.TypeInitializationException: The type initializer for 'System.ConsoleDriver' threw an exception. ---> System.Exception: Magic number is wrong: 542
  at System.TermInfoReader.ReadHeader (System.Byte[] buffer, System.Int32& position) [0x00028] in <a84b655e5e6a49ee96b338ec792f5580>:0 
  at System.TermInfoReader..ctor (System.String term, System.String filename) [0x0005f] in <a84b655e5e6a49ee96b338ec792f5580>:0 
  at System.TermInfoDriver..ctor (System.String term) [0x00055] in <a84b655e5e6a49ee96b338ec792f5580>:0 
  at System.ConsoleDriver.CreateTermInfoDriver (System.String term) [0x00000] in <a84b655e5e6a49ee96b338ec792f5580>:0 
  at System.ConsoleDriver..cctor () [0x0004d] in <a84b655e5e6a49ee96b338ec792f5580>:0 
   --- End of inner exception stack trace ---
  at System.Console.SetupStreams (System.Text.Encoding inputEncoding, System.Text.Encoding outputEncoding) [0x00007] in <a84b655e5e6a49ee96b338ec792f5580>:0 
  at System.Console..cctor () [0x0008e] in <a84b655e5e6a49ee96b338ec792f5580>:0 
   --- End of inner exception stack trace ---
  at System.Diagnostics.Process.StartWithCreateProcess (System.Diagnostics.ProcessStartInfo startInfo) [0x00337] in <640558357cc84390be8186623166329d>:0 
  at System.Diagnostics.Process.Start () [0x0003a] in <640558357cc84390be8186623166329d>:0 
  at (wrapper remoting-invoke-with-check) System.Diagnostics.Process:Start ()
  at System.Diagnostics.Process.Start (System.Diagnostics.ProcessStartInfo startInfo) [0x0001b] in <640558357cc84390be8186623166329d>:0 
  at Duplicati.Library.AutoUpdater.UpdaterManager.RunFromMostRecentSpawn (System.Reflection.MethodInfo method, System.String[] cmdargs, Duplicati.Library.AutoUpdater.AutoUpdateStrategy defaultstrategy) [0x00196] in <bea267acd22e46748297665d861d8f02>:0 
  at Duplicati.Library.AutoUpdater.UpdaterManager.RunFromMostRecent (System.Reflection.MethodInfo method, System.String[] cmdargs, Duplicati.Library.AutoUpdater.AutoUpdateStrategy defaultstrategy) [0x0001b] in <bea267acd22e46748297665d861d8f02>:0 
  at Duplicati.CommandLine.Program.Main (System.String[] args) [0x0001a] in <cce58b5528b54025a8c535610dc1cdaf>:0 
```

**Comment by TekkiWuff** [comment]
This might be related to the following bug in mono that has been fixed a few days ago and will be in mono 5.12: https://github.com/mono/mono/issues/6752.

Maybe try the workaround described in this comment here: https://github.com/mono/mono/issues/6752#issuecomment-365212655

**Comment by blacky14** [comment]
@TekkiWuff thank you very much, I should have googled the generic error message, not only search in the duplicati issues! :-)

the workaround also works, so I'm closing this ticket.

thanks again!
